c++/src/kj/CMakeLists.txt

```diff
@@ -78,8 +78,7 @@ set(kj-std_headers
 )
 add_library(kj ${kj_sources})
 add_library(CapnProto::kj ALIAS kj)
-# TODO(cleanup): Use cxx_std_14 once it's safe to require cmake 3.8.
-target_compile_features(kj PUBLIC cxx_generic_lambdas)
+target_compile_features(kj PUBLIC cxx_std_14)
```

Comment:
We actually require C++17 these days (on the master branch; even newer on v2).

Reply:
Done.